miden-vm/tests/integration/cli/data/adv_map.masm

```diff
@@ -0,0 +1,26 @@
+adv_map.A=2
+adv_map.B(0x0200000000000000030000000000000004000000000000000500000000000000)=3
+adv_map.C([3,4,5,6])=4
```

Comment:
Question: how do we write a sequence of field elements for the advice map entry? For example, If I wanted to have a sequence of 6 elements, would I write this as:
```
adv_map.A=2,3,4,5,6,7
```
or
```
adv_map.A=[2,3,4,5,6,7]
```
Or something else?

Reply:
I see. Comma-separated is better, but maybe we can still use something more obvious - e.g., curly braces?
```
adv_map.A={ 2, 3, 4, 5, 6, 7 }
```
Though, square brackets would look better, I think.

cc @Al-Kindi-0 @adr1anh @bitwalker.

Reply:
Agree that square brackets is best but curly brackets would also work fine in my opinion